capstone/src/main/webapp/signUp.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Gets the information from the textboxes to send to datastore.
+ */
+function sendSignUpInfo() {
+  const name = document.getElementById('name').value;
+  const cuisine = document.getElementById('cuisine').value;
+  const story = document.getElementById('story').value;
+  const phone = document.getElementById('phone').value;
```

Comment:
I might add some sort of defensive check to make sure document.getElementById('sign-up') is not null before proceeding with the rest of the getElementById calls, just as a small precaution.
Also, you could maybe do something like 
const name = document.getElementById('name').value || '';
to make sure that we pass an empty string instead of null in case the user leaves something blank.

Reply:
I looked into the types and how the value works and it apparently sends back just an empty string (so not equal to null).

Reply:
In order to avoid this I just added a text box that appears if you don't fill one of the text areas. Thus not calling the server if we don't have all the information.

Reply:
Looks great!